src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
So, at boot or resync:

1. Build array of expected cert hashes
1. Unlink existing regular files with unexpected hashes
1. Check file size of expected files against config contents to avoid unnecessary reading of the certs during boot, and rewrite any files with bad sizes
1. Leave symlinks alone unless there is a name collision

When editing a CA:
* Just rewrite the file

When deleting a CA:
* Just unlink the file